modules/openapi-generator/src/main/resources/php/ObjectSerializer.mustache

```diff
@@ -339,4 +339,29 @@ class ObjectSerializer
             return $instance;
         }
     }
+    
+    /**
+	 * Verify the Data structure to make sure all the objects are validated
+	 * @param mided $data object to be validated
+	 * @throws \InvalidArgumentException
+	 */
+	public static function verifyDeserializedData ($data) {
```

Comment:
Not sure about the design here: the data needs to be valid enough for the deserialization to take place. If that already happened, `ObjectSerializer` has nothing to do with it.